File: week02-210627-210703/moregorenine/baekjoon/step3_for/q1_2739/main.js
const fs = require('fs')
const stdin = (process.platform === 'linux'
        ? fs.readFileSync(0, "utf8")
        : `2`
)

const number = parseInt(stdin)

function displayMultiplicationTable(number, count) {
    for (let i = 1; i <= count; i++) {
        console.log(`${number} * ${i} = ${number * i}`);
    }
}

displayMultiplicationTable(number, 9)

File: week02-210627-210703/moregorenine/baekjoon/step3_for/q1_2739/main.py
import sys

# inHour, inMinute = map(int, sys.stdin.readline().split())

number = int(sys.stdin.readline())


def displayMultiplicationTable(number, count):
    for i in range(1, count + 1):
        print(f'{number} * {i} = {number * i}')


displayMultiplicationTable(number, 9)
